apps/obs-web/src/views/DebrieferTrips.vue

```diff
@@ -7,6 +7,7 @@
       :simple="false"
       uniqueKey="_id"
       :enableSelection="true"
+      :showSelectionBoxes="true"
```

Comment:
does this hide the leftmost column of checkboxes?

Reply:
yes added this for the biospecimens page. When a user clicks on a link in the catch page want the corresponding biospecimens to be highlighted without the checkboxes. Highlighting on the trips and hauls page filters the results, highlighting on biospecimens has no functionality, it doesn't filter so wanted to remove the checkboxes making it clear nothing will happen when a record is highlighted.

Reply:
nice!  that will be handy!